dallinger/experiment_server/experiment_server.py

```diff
@@ -527,14 +527,6 @@ def prepare_advertisement():
     hit_id = entry_data.get("hit_id")
     assignment_id = entry_data.get("assignment_id")
     worker_id = entry_data.get("worker_id")
-    prolific_pid = entry_data.get("PROLIFIC_PID")
```

Comment:
This code failed originally because `normalize_entry_information` `pop`s these terms from the entry data, so they are no longer available by this point. It's OK though because `normalize_entry_information` maps the terms to `assignment_id`, `hit_id`, and `worker_id` already.

Reply:
OK